### PR TITLE
Issue credits backend

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/api-calls/credits-requests.js [hunk 1]
```diff
@@ -1,17 +1,65 @@
 import axios from "axios";
 import handleError from "./handle-errors";
 
-const CREDITES_REQUESTS_BASE = "/credites-requests";
+const CREDITES_REQUESTS_BASE = "/credits-requests";
```
Comment (thejoefriel) [inline client/src/api-calls/credits-requests.js:4]: can you fix this type to make it CREDITS_REQUESTS_BASE

File: client/src/pages/Staff/CreditRequests/index.js [hunk 2]
```diff
@@ -17,22 +19,45 @@ import {
...
+  getPendingCreditsRequests,
+  getPreviousCreditsRequests,
+  selectCreditsRequest,
+  pendingRequests,
+  pendingRequestsLoaded,
+  prevRequests,
+  prevRequestsLoaded,
+
+  // TODO error handling
+  // prevError,
```
Comment (thejoefriel) [inline client/src/pages/Staff/CreditRequests/index.js:32]: Is this commented code something that should be done? Do we have an issue for it?

File: client/src/api-calls/credits-requests.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/api-calls/credits-requests.js:4]: done

File: client/src/pages/Staff/CreditRequests/index.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/Staff/CreditRequests/index.js:32]: i think this need someone to go through the app and make a consistent loading indicator and error messages

**Pull request merged**